db/schema.rb

```diff
@@ -66,7 +66,7 @@
 
   create_table "members", force: :cascade do |t|
     t.text "name"
-    t.text "email"
+    t.text "email", null: false
```

Comment:
This schema file is the result of running migrations. It also shows the current state of the entire data model, so you can review it to get a quick sense of what things are like.